_posts/2022-07-15-ia-update-4.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "Uncovering What You Can’t See: Innovation Adoption Workshop Series Asks, ‘Is Your Culture Ready for Change?’"
+layout: post
+initiative: Innovation Adoption
+date: July 15, 2022
+---
+By the Innovation Adoption Practice 
+
+<em>In June 2022, the CoE Innovation Adoption (IA) practice published a <a href="{{site.baseurl}}/docs/CulturePreAssessmentGuide.pdf" target="_blank">Culture Pre-Assessment Guide</a> and hosted a workshop series to highlight how elements of culture can help or hinder an agency’s IT modernization initiative and exposed participants to the tools that can be used to conduct a culture pre-assessment. Over 50 federal employees from nearly 20 different organizations were in attendance.</em>
+
+<a href="{{site.baseurl}}/images/gsagraphic.jpg" target="_blank" ><img src="{{site.baseurl}}/images/gsagraphic.jpg"></a>
```

Comment:
[question] is there a reason to make this a link?

[issue] for accessibility, this image should have alt text